modules/runner/src/main/java/org/apache/ignite/internal/app/IgnitionImpl.java

```diff
@@ -138,6 +140,24 @@ public void stop(String nodeName) {
         });
     }
 
+    /**
+     * Stops all Ignite instances started in this JVM.
+     */
+    @TestOnly
+    public void stopAll() {
+        List<String> nodeNames = new ArrayList<>(nodes.keySet());
+
+        if (!nodeNames.isEmpty()) {
+            LOG.info("Going to stop the following Ignite instances: " + nodeNames);
+
+            for (String nodeName : nodeNames) {
+                stop(nodeName);
+            }
+
+            LOG.info("Stopped the following Ignite instances: " + nodeNames);
```

Comment:
Do we really need this message as well?

Reply:
Stopping of an Ingite instance might hang. It seems helpful to see the 'Everything is finished' to make sure that nothing has hung.